src/net_processing.cpp

```diff
@@ -335,6 +347,10 @@ class PeerManagerImpl final : public PeerManager
       * on extra block-relay-only peers. */
     bool m_initial_sync_finished{false};
 
+    /** After connecting to a peer, wait this amount of time before considering
+     * disconnection based on inactivity */
+    const int64_t m_peer_connect_timeout;
```

Comment:
I'd prefer if this was a chrono duration, but I understand the context of keeping it consistent with net.

Reply:
I've now removed this again. Sorry :grimacing: